redcap_bridge/project_building.py

```diff
@@ -160,6 +164,8 @@ def combine_values(value1, value2):
     if output_file is not None:
         combined_df.to_csv(output_file, index=False)
 
+    print("\nCustomizing", len(customization_df) , "fields\n", end='')
```

Comment:
As an extra feature maybe it would make sense to print also the names of the customized fields (if there are not too many). Or we print the list of names and shorten after 100 characters... @killianrochet What do you think?

Reply:
Ok, I'll do some tests to see if there are not too many fields and that it remains readable !